CODEOWNERS

```diff
@@ -47,31 +47,18 @@ homeassistant/components/*/zwave.py @home-assistant/z-wave
 homeassistant/components/hassio/* @home-assistant/hassio
 
 # Individual platforms
-homeassistant/components/notify/aws_lambda.py @robbiet480
-homeassistant/components/notify/aws_sns.py @robbiet480
-homeassistant/components/notify/aws_sqs.py @robbiet480
-homeassistant/components/notify/file.py @fabaff
```

Comment:
Please double check that you properly re-distributed these, I think at least `tts.amazon_polly` is missing.